examples/helloworld/hello.js

```diff
@@ -6,9 +6,10 @@ Promise.all([System.import('pdfjs/display/api'),
              System.import('pdfjs/display/network'),
              System.resolve('pdfjs/worker_loader')])
        .then(function (modules) {
-  var api = modules[0], global = modules[1];
+  var api = modules[0], global = modules[1], network = modules[2];
   // In production, change this to point to the built `pdf.worker.js` file.
   global.PDFJS.workerSrc = modules[3];
+  api.setPDFNetworkStreamClass(network.PDFNetworkStream)
```

Comment:
Please add a semi-colon at the end of this line. Also, let's move this line to just below https://github.com/mozilla/pdf.js/pull/8874/files#diff-073ee7f864077e4b3968f8df3ba154e9R9 instead.